test/EndToEnd/tests/BuildIntegratedTest.ps1

```diff
@@ -690,4 +690,12 @@ function Test-BuildIntegratedVSandMSBuildNoOp {
 
     #Assert
     Assert-True ($MsBuildRestoreTimestamp -eq $VSRestoreTimestamp)
+}
+
+function BuildProjectTemplateTestCases([string[]]$ProjectTemplates) {		
```

Comment:
Was this something that used to be a part of lsltests.ps1 or?

Reply:
yes, that's correct . see here: https://github.com/NuGet/NuGet.Client/commit/5f3fd208c5b6eb09fad3bbf27c7fa46a66d46b57#diff-ddd46277d18da3a2974b840e2ca7dcdcL246